website/docs/d/oauth_token.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_oauth_token"
+sidebar_current: "docs-tfe-datasource-oauth-token"
+description: |-
+  Use this datasource to get the ID of the OAuth token of an assocaited VCS configuration.
+---
+
+# tfe_oauth_token
```

Comment:
I don't see a Data Source for this?

Reply:
Correct, this should not be in this commit. Fixed in the next commit.